core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -3378,7 +3378,7 @@ XDDkwd__(SUBQUERY_UNNESTING,			"ON"),
   DDkwd__(TRAF_READ_OBJECT_DESC,                       "OFF"),   
 
   DDkwd__(TRAF_RELOAD_NATABLE_CACHE,                   "OFF"),
-  DD_____(TRAF_SAMPLE_TABLE_LOCATION,                  "/sample/"),
```

Comment:
Would be good to come up with a different location rather than sample.  The name 'sample' connotes to kind of example table which might be deleted unknowingly.